### PR TITLE
Add jsonp multiple params support

### PR DESCRIPTION
**Pull request by orenmagen100**
support params in jsonp requests

### CODE REVIEW

File: src/leonardo/configuration.srv.ts [hunk 1]
```diff
@@ -124,8 +144,10 @@ export function leoConfiguration() {
       return state.jsonpCallback;
     }
 
-    const postfix = state.url.split('callback=')[1];
-    return postfix.split('&')[0];
+    if (state.url.indexOf('callback=') > 1) {
+      let postfix = state.url.split('callback=')[1];
```
Comment (tsachis) [inline src/leonardo/configuration.srv.ts]: const

File: src/leonardo/configuration.srv.ts [hunk 2]
```diff
@@ -61,13 +61,33 @@ export function leoConfiguration() {
   function activeJsonpState(state, callbackName: string) {
     const funcName = state.name + callbackName;
     if (_jsonpCallbacks[funcName]) return;
-    if (typeof window[callbackName] === 'function') {
-      _jsonpCallbacks[funcName] = window[callbackName];
-      window[callbackName] = dummyJsonpCallback;
+
+    let evaluatedFunction = eval(callbackName);
```
Comment (tsachis) [inline src/leonardo/configuration.srv.ts]: const

Comment (orenmagen100) [inline src/leonardo/configuration.srv.ts]: fixed

File: src/leonardo/configuration.srv.ts (hunk 1 above)
Comment (orenmagen100) [inline src/leonardo/configuration.srv.ts]: fixed

**Comment by orenmagen100** [comment]
@tsachis  Fixed comments and Deactivate. Plz review

**Pull request merged**